### PR TITLE
exploration and pagestats

### PR DESCRIPTION
**Pull request by JosuaKrause**
update exploration response and pagestats

### CODE REVIEW

File: public/js/main.js [hunk 1]
```diff
@@ -12,6 +12,33 @@ if ('serviceWorker' in navigator) {
   });
 }
 
+export function shortStringAsNum(text) {
```
Comment (myjyby) [inline public/js/main.js:15]: can this be a String.prototype.extension?

File: routes/browse/contributors/render.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const { page_content_limit, modules, metafields, lazyload, map, DB } = include('config/')
 const load = require('./load/')
-const { array, checklanguage, datastructures } = include('routes/helpers/')
+const { array, checklanguage, datastructures, pagestats } = include('routes/helpers/')
```
Comment (myjyby) [inline routes/browse/contributors/render.js:3]: not sure why we need pagestats on contributor page.

Comment (JosuaKrause) [reply routes/browse/contributors/render.js:3]: team

**Review by myjyby — APPROVED** [review]
lgtm

**Pull request merged**